src/main/java/com/example/javacohort3/ZipCodeBank/domains/Deposit.java

```diff
@@ -11,7 +14,8 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    private Enum type;
+    private TransactionMedium type;
+    private TransactionStatus types;
```

Comment:
Why would you name the TransactionStatus field "types"?

Reply:
Why would you also name the TransactionMedium field "type"? This naming convention is too confusing.
Also why did you remove the TransactionType field ??????

Reply:
I will change those but I was told to remove the TransactionType field so I removed it